src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/GetPileupSummaries.java

```diff
@@ -81,6 +82,9 @@
 
     private VariantContext lastVariant = null;
 
+    private boolean sawVariantsWithoutAlleleFrequency = false;
```

Comment:
Can you update the tool docs to state that AF must appear in every variant in the VCF?

Are you sure you want to throw an exception if a variant is missing allele frequency?  Not simply output some NAs?  You know the use case better than I do, so this suggestion can be ignored.

Reply:
@LeeTL1220 If only some AFs are missing no error is thrown.  I added a test that verifies this.